### DISCUSSION TITLE
Plugins for swc?

### DISCUSSION THREAD

**Discussion by jvanderen1** [ben-rogerson/twin.macro#520] [Ideas]
I would like to know if there are any plugins in the works to use twin.macro with swc?

https://swc.rs/

**Comment by ben-rogerson** [comment]
There aren't any ported versions of Twin for swc yet.

[Check out the existing discussion on this → ](https://github.com/ben-rogerson/twin.macro/discussions/516)

**Reply by jvanderen1** [reply]
Oh wow, I just searched this yesterday and didn't see this discussion posted recently. Exciting stuff. Thanks!

**Reply by arlyon** [reply]
For anyone visiting this page, I left a comment on the linked post about an experimental swc plugin I am writing called stailwc, and am looking for contributors: https://github.com/arlyon/stailwc
